app/controllers/security_group_controller.rb

```diff
@@ -14,4 +14,261 @@ def self.display_methods
   end
 
   menu_section :net
+
+  def button
+    @edit = session[:edit] # Restore @edit for adv search box
+    params[:display] = @display if %w(vms instances images).include?(@display)
+    params[:page] = @current_page unless @current_page.nil? # Save current page for list refresh
+
+    @refresh_div = "main_div"
+
+    case params[:pressed]
+    when "security_group_tag"
+      return tag("SecurityGroup")
+    when 'security_group_delete'
+      delete_security_groups
+    when "security_group_edit"
+      checked_security_group_id = get_checked_security_group_id(params)
```

Comment:
Same comment as in https://github.com/ManageIQ/manageiq/pull/12097/files#r92808587 .. please use `include Mixins::CheckedIdMixin` and call `checked_item_id(params)` here, instead of reimplementing.